document/src/main/kotlin/com/ritense/document/event/DocumentAssigned.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.document.event
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.ritense.outbox.domain.BaseEvent
+
+class DocumentAssigned(documentId: String, documentContent: ObjectNode) : BaseEvent(
```

Comment:
Alternatively, this could accept a document as the arguments and then take the ID and content from the document.

Reply:
I've thought of that, but that was not would not match the style of the events already there, and and we'd need an `ObjectMapper` in the event class. We can still refactor it later on if we change our mind.